tools/test.py

```
@@ -0,0 +1,59 @@
+# author = next(scholarly.search_author('Steven A Cholewiak'))
+# scholarly.pprint(author)
+# search = get_arxiv('all:cervical cancer image segmentation', 2000)
+# rst = filter_arxiv(search, 'cervical cancer cell segmentation',end_time=datetime.datetime(2019, 1, 1, 0, 0, 0, tzinfo=datetime.timezone.utc))
+# ref_eval = arxiv2ref(rst)
+
+
+ref_eval = get_google('cervical cancer image segmentation', year_high=2018, topk=10)
+
+avg_cite = get_avg_cite_num(ref_eval, year_high=2018)
+
+ref_text = extract_ref(r"C:\Users\Ocean\Downloads\1811.00849.pdf")
```

Ruff (0.8.2)

`extract_ref` may be undefined, or defined from star imports

(F405)